codejail/safe_exec.py

```diff
@@ -117,69 +118,13 @@ def flush(self, *args, **kwargs):
         # Execute the sandboxed code.
         """
         exec(code, g_dict)
+        """))
+
+    the_code.append(inspect.getsource(json_safe))
```

Comment:
Now that I see this line here, it occurs to me that this means we have to have the source in our deploys.  We do have source today, but we should make sure to in the future.  We should add `zip_safe=False` to the setup.py.